src/charts/stacked-bar.js

```diff
@@ -408,6 +408,10 @@ define(function(require){
         function drawGridLines() {
             let scale = isHorizontal ? xScale : yScale;
 
+            svg.select('.grid-lines-group')
+                .selectAll('line')
```

Comment:
Could we turn this into a util/helper function? It will most likely be used in more charts that will be built in the future.

Reply:
Something like:

```function cleanGridlines() { //code }```

Reply:
```function removeGridLines(svg) {
     svg.select('.grid-lines-group').selectAll('line').remove()
}

Reply:
Yeah!
We probably should refactor all the grid line logic into one helper like the axis one.
I will create a ticket for it!